src/js/utils/styles.js

```diff
@@ -157,6 +157,8 @@ export const focusStyle = css`
         normalizeColor(props.theme.global.focus.border.color, props.theme)}
       solid 2px;
   }
+  outline-color: ${props =>
+    normalizeColor(props.theme.global.focus.border.color, props.theme)};
```

Comment:
On the storybook 'Clickable' example, the focus isn't shown per row, it focuses on the whole Table after the alert is shown, not sure if this is the expected behavior.
![image](https://user-images.githubusercontent.com/6320236/64032493-cdd4dd80-cb07-11e9-927d-2fc9155e188d.png)

Reply:
Correct. I think this makes it easier to traverse across the table to another control. For instance, if there is a footer, you would have to tab through all of the rows to get to it. This way, you can tab to the table and then arrow to the row you want.